ci/launch_cluster.sh

```diff
@@ -2,34 +2,56 @@
 
 set -x -e -o pipefail
 
+# Ensure jq is installed.
+if ! command -v jq >/dev/null; then
+    echo "jq was not found. Please install it."
+fi
+
 # Two parameters are expected: CHANNEL and VARIANT where CHANNEL is the respective PR and
 # VARIANT could be one of four custer variants: open, strict, permissive and disabled
-if [ "$#" -ne 2 ]; then
-    echo "Expected 2 parameters: <channel> and <variant> e.g. launch_cluster.sh testing/pull/1739 open"
+if [ "$#" -ne 3 ]; then
+    echo "Expected 3 parameters: launch_cluster.sh <channel> <variant> <deployment-name>"
+    echo "e.g. CLI_TEST_SSH_KEY="test.pem" launch_cluster.sh testing/pull/1739 open si-testing-open"
     exit 1
 fi
 
 CHANNEL="$1"
 VARIANT="$2"
+DEPLOYMENT_NAME="$3"
 
 if [ "$VARIANT" == "open" ]; then
   TEMPLATE="https://s3.amazonaws.com/downloads.dcos.io/dcos/${CHANNEL}/cloudformation/multi-master.cloudformation.json"
 else
   TEMPLATE="https://s3.amazonaws.com/downloads.mesosphere.io/dcos-enterprise-aws-advanced/${CHANNEL}/${VARIANT}/cloudformation/ee.multi-master.cloudformation.json"
 fi
 
-echo "Workspace: ${WORKSPACE}"
 echo "Using: ${TEMPLATE}"
 
-PLATFORM=`uname`
-if [ "$PLATFORM" == 'Darwin' ]; then
-    brew install gettext
-else
-    apt-get update && apt-get install -y -t jessie-backports gettext-base wget
+# Ensure envsubst is available.
+if ! command -v envsubst >/dev/null 2>&1; then
```

Comment:
I would vote against on installing missing dependencies in this file. I am in favour of checking the environment and failing early, like the `jq` requirement above.

For this purpose you could introduce a `install_deps.sh` file that the CI will run first to check the env **and** install missing deps.

Reply:
Done.